edb/edgeql/parser/grammar/ddl.py

```diff
@@ -1775,6 +1775,9 @@ def reduce_POSTFIX(self, *kids):
     def reduce_PREFIX(self, *kids):
         self.val = kids[0].val
 
+    def reduce_NARY(self, *kids):
```

Comment:
ternary?

Reply:
We can add N-ary when we need to, but if/else it ternary, right?

Reply:
Yes, but ternary and n-ary are the same from the standpoint of handling them, so it makes little sense in having a separate kind for every value of N.

Reply:
Arent' binary and ternary and n-ary all the same from the standpoint of handling them? ;)